src/main/java/frc/robot/subsystems/intake/IntakeCal.java

```diff
@@ -0,0 +1,28 @@
+package frc.robot.subsystems.intake;
+
+import edu.wpi.first.math.controller.SimpleMotorFeedforward;
+import frc.robot.Constants;
+
+public class IntakeCal {
+
+  /** PID values for the intake pivot motor */
+  public static final double INTAKE_PIVOT_P = Constants.PLACEHOLDER_DOUBLE,
+      INTAKE_PIVOT_I = Constants.PLACEHOLDER_DOUBLE, INTAKE_PIVOT_D = Constants.PLACEHOLDER_DOUBLE;
+
+  /** max velocity and acceleration (deg per second) for intake pivot motor */
+  public static final double PIVOT_MAX_VELOCITY_DEG_PER_SECOND = Constants.PLACEHOLDER_DOUBLE,
+      PIVOT_MAX_ACCELERATION_DEG_PER_SECOND_SQUARED = Constants.PLACEHOLDER_DOUBLE;
+
+  /** intake positions (degrees) */
+  public static final double INTAKE_DEPLOYED_POSITION_DEGREES = Constants.PLACEHOLDER_DOUBLE, INTAKE_STOWED_POSITION_DEGREES = Constants.PLACEHOLDER_DOUBLE, INTAKE_SAFE_POSITION_DEGREES = Constants.PLACEHOLDER_DOUBLE;
+
+  public static final double INTAKE_MARGIN_DEGREES = Constants.PLACEHOLDER_DOUBLE, CONVEYOR_ZONE_THRESHOLD_DEGREES = Constants.PLACEHOLDER_DOUBLE;
+
+  /** intake power [-1.0,1.0] */
+  public static final double INTAKING_POWER = Constants.PLACEHOLDER_DOUBLE, REVERSE_INTAKING_POWER = Constants.PLACEHOLDER_DOUBLE;
+
+  /** intake feedforward */
+  public static final SimpleMotorFeedforward INTAKE_PIVOT_FEEDFORWARD = new SimpleMotorFeedforward(Constants.PLACEHOLDER_DOUBLE,Constants.PLACEHOLDER_DOUBLE); 
```

Comment:
The values passed into `SimpleMotorFeedforward` should be their own variables.

Reply:
You didn't actually fix this, you're still passing placeholders directly into the `SimpleMotorFeedforward`.